### ISSUE TITLE
Add `expiresAt` to RFQ

### ISSUE THREAD

**Issue by decentralgabe** [TBD54566975/tbdex#267]
Similar to the [Quote Message](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#quote) it may be useful to add an `expiresAt` property to the [RFQ Message](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#rfq-request-for-quote). This would prevent potential old or already fulfilled quotes from being responded to causing unnecessary traffic and/or confusion.

Separately, I'd like to ask if the `expiresAt` property should be defined in either message payload at all, or should instead be a required field in the JWT payloads we create.

**Comment by phoebe-lew** [comment]
What does an RFQ expiring mean in terms of the user story? Are you thinking - after a certain period of time, if the PFI has not responded to the RFQ, there is no point in the PFI sending a quote?

>  This would prevent potential old or already fulfilled quotes from being responded to causing unnecessary traffic
I'm not following the connection between already fulfilled quotes and RFQs expiring

**Comment by decentralgabe** [comment]
> Are you thinking - after a certain period of time, if the PFI has not responded to the RFQ, there is no point in the PFI sending a quote?

Yes. The user could say 'I'm asking 5 PFIs for quotes and will pick the best within the next 3 minutes'

> I'm not following the connection between already fulfilled quotes and RFQs expiring

Following what I said above, a user could've chosen to go with PFI A and an expiry would prevent PFI B from unnecessarily responding outside of that time window.
